README.md

```diff
@@ -1,64 +1,1409 @@
+tidbcloud_backup.example_backup: Destroying... [id=1350048]
+tidbcloud_backup.example_backup: Destruction complete after 2s
+tidbcloud_cluster.example_cluster: Destroying... [id=1379661944630234067]
+tidbcloud_cluster.example_cluster: Destruction complete after 0s
+╷
+│ Warning: Unsupported
+│ 
+│ restore can't be delete
+╵
+
+Destroy complete! Resources: 4 destroyed.
+```
+
+Note that a warning is appeared for restore can't be deleted.
+
+If you execute `terraform show`, you will find nothing for all the states is cleared:
+```
+$ terraform show
+
+```
 
 ## Developing the Provider
 
+### Environment
+
 If you wish to work on the provider, you'll first need [Go](http://www.golang.org) installed on your machine (see [Requirements](#requirements) above).
 
-To compile the provider, run `go install`. This will build the provider and put the provider binary in the `$GOPATH/bin` directory.
+### Building the provider from source
 
-To generate or update documentation, run `go generate`.
+1. Clone the repository
+```shell
+git clone git@github.com:tidbcloud/terraform-provider-tidbcloud
+```
+2. Enter the repository directory
+```shell
+cd terraform-provider-tidbcloud
+```
+3. Build the provider using the Go `install` command. This will build the provider and put the provider binary in the `$GOPATH/bin` directory.
+```shell
+go install
+```
 
-In order to run the full suite of Acceptance tests, run `make testacc`.
+### Generate or update documentation in docs file
 
-*Note:* Acceptance tests create real resources, and often cost money to run.
+run `go generate`
 
-```shell
-make testacc
+### Running the acceptance test
+
+see [here](./internal/README.md) for more detail.
+
+### Debug the provider
+
+I will introduce how to debug with Terraform CLI development overrides. About other ways to debug the provider. See [terraform official doc](https://www.terraform.io/plugin/debugging) for more detail
+
+Development overrides is a method of using a specified local filesystem Terraform provider binary with Terraform CLI, such as one locally built with updated code, rather than a released binary.
+
+1. create `.terraformrc` in your operating system user directory
+```
+provider_installation {
+
+  dev_overrides {
+      "hashicorp.com/edu/tidbcloud" = "/usr/local/go/bin"
+  }
+
+  # For all other providers, install them directly from their origin provider
+  # registries as normal. If you omit this, Terraform will _only_ use
+  # the dev_overrides block, and so no other providers will be available.
+  direct {}
+}
+```
+
+2. run go install in the terraform-provider-tidbcloud, you will find the `terraform-provider-tidbcloud` will be installed under the `/usr/local/go/bin` 
 ```
+go install
+```
+
+3. Terraform CLI commands, such as terraform apply, will now use the specified provider binary if you follow the below config:
+```
+terraform {
+  required_providers {
+    tidbcloud = {
+      source = "hashicorp/tidbcloud"
```

Comment:
hashicorp/tidbcloud?

Reply:
we need not to use terraform-provider-tidbcloud, tidbcloud is enough. This is stated in the terraform documentation， but I can not find it now